### ISSUE TITLE
About the network design?

### ISSUE THREAD

**Issue by he010103** [ybsong00/CREST-Release#7]
@ybsong00 I noticed that the CREST only use 22 layers of VGG. By using all the convolution layers, will the accuracy increase? And the learning rate of the residual network is too slow. Does it mean that the parameter of the network need not to be changed substantially. Therefore, can the network be trained offline with large tracking benchmark, thus making the CREST faster without much online training?

**Comment by ybsong00** [comment]
@he010103 Thanks for your interest.

"By using all the convolution layers, will the accuracy increase? "

It depends on the feature discriminative ability on the benchmark tasks and how well it is integrated with other tracker components (e.g., DCF or CNN). Currently, it is only empirically configured without a theoretical explanation. In this sense, the accuracy improvement is based on multiple trials of the feature extraction and response generation.

"And the learning rate of the residual network is too slow. Does it mean that the parameter of the network need not be changed substantially?"

There are several updating strategies and ours is to update the network frequently with small steps. It decreases the effect of online noisy data update to prevent overfitting to some extent. 

"Therefore, can the network be trained offline with large tracking benchmark, thus making the CREST faster without much online training?"

Generally, there are two types of strategies. The first one is SiamFC to pretrain model using millions of imagenet vidoes offline and directly use it for testing. This is fast but the accuracy is not guaranteed,  especially when test sequence varies much from the training ones. The second one is ours to let network adapt to each test sequence. This is accurate but slow since online updating costs much time. A clever way is to combine these two strategies like what MDNet does, pretaining a model offline and update it online. I do believe it will accelerate the CREST through pretraining the base layer and without much online finetuning since currently all the layers are randomly initialized.
